src/main/java/jenkins/plugins/logstash/LogstashConfiguration.java

```diff
@@ -53,6 +58,38 @@ public void setEnableGlobally(boolean enableGlobally)
     this.enableGlobally = enableGlobally;
   }
 
+  public boolean isMilliSecondTimestamps()
+  {
+    return milliSecondTimestamps;
+  }
+
+  public void setMilliSecondTimestamps(boolean milliSecondTimestamps)
+  {
+    this.milliSecondTimestamps = milliSecondTimestamps;
+    createDateFormatter();
+  }
+
+  public FastDateFormat getDateFormatter()
```

Comment:
I think this needs to be synchronized

Reply:
With the constants it doesn't need to be synchonized.